src/main/java/uk/gov/ons/ctp/response/action/service/ActionProcessingService.java

```diff
@@ -58,7 +58,7 @@ public ActionProcessingService(ActionRequestDecorator[] decorators) {
    *
    * @param action the action to deal with
    */
-  @Transactional
+  @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
I suspect that the Action passed to this method, because it was from a _different_ transactions scope, could cause problems... but I shall test

Reply:
Removing the unneeded transaction from higher up fixed this